src/tools/rbd/Utils.h

```diff
@@ -16,6 +16,24 @@ namespace utils {
 
 static const std::string RBD_DIFF_BANNER ("rbd diff v1\n");
 
+static const std::string RBD_IMAGE_BANNER_V2 ("rbd image v2\n");
+static const std::string RBD_IMAGE_SNAPS_BANNER_V2 ("rbd image snaps v2\n");
```

Comment:
Nit: instead of "snaps", perhaps just "diffs" since this delineates the start of the diff section (and included the HEAD revision which is technically not a snapshot).